gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/OrganizationLicenseRepository.java

```diff
@@ -21,4 +21,8 @@
 
 public interface OrganizationLicenseRepository extends CrudRepository<OrganizationLicense, String> {
     Optional<OrganizationLicense> findByOrganizationId(String organizationId) throws TechnicalException;
+
+    OrganizationLicense create(OrganizationLicense organizationLicense) throws TechnicalException;
```

Comment:
I don't believe this create + update method are necessary since they are already implemented in the CrudRepository class.

Reply:
Yes you're right we can remove them